md2all.bat

```diff
@@ -5,8 +5,22 @@ pushd source
 set front=front-matter.md
 set content=chapter1.md chapter2.md chapter3.md chapter4.md chapter5.md chapter6.md chapter7.md chapter8.md bonus-chapter.md back-matter.md
 
+@REM Create folders:
+if not exist ../output mkdir ../output
+if not exist ../output/web mkdir ../output/web
+
 @REM Build HTML:
-pandoc -s %front% %content% -o ../output/ebh.html --toc --section-divs --lua-filter ../styling/pagebreak.lua --lua-filter ../styling/html-open-links-in-new-tab.lua --lua-filter ../styling/html-lazy-images.lua --template ../styling/html-template.html
+pandoc --shift-heading-level-by -1 --split-level 1 -t chunkedhtml -s %front% %content% -o ../output/web/ebh --toc --section-divs --lua-filter ../styling/pagebreak.lua --lua-filter ../styling/html-open-links-in-new-tab.lua --lua-filter ../styling/html-lazy-images.lua --template ../styling/html-template.html
```

Comment:
Why do we need `--shift-heading-level-by 1`?
Isn't that what's causing the issue with h1, h2, etc. that required you to change the styles of those tags?

Reply:
I forgot to remove this parameter. I added it while trying to fix the issue with headings, but it did not work. I am pushing a commit removing it now.